evaluation/segmentationevaluation.py

```diff
@@ -0,0 +1,142 @@
+import argparse
+import csv
+import datetime
+import glob
+import os
+import random
+
+import h5py
+import napari
+
+from evaluation import RESULTS
+from evaluation.makestatistics import compute_stats
+
+KEYS_TO_USE = {'marker': {'data_type': 'image', 'visible': False},
+               'nuclei': {'data_type': 'image', 'visible': True},
+               'serum': {'data_type': 'image', 'visible': False},
+               'cell_segmentation': {'data_type': 'label', 'visible': True},
+               'mask': {'data_type': 'label', 'visible': False}
+               }
+
+
+def write_csv(csv_path, dataset, name, value):
+    with open(csv_path, 'a+') as f:
+        writer = csv.writer(f)
+        writer.writerow([dataset, name, value])
+
+
+def load_stack(h5_path):
+    stack = {}
+    with h5py.File(h5_path, "r") as f:
+        for key, value in KEYS_TO_USE.items():
+            stack[key] = f[key][list(f[key].keys())[-1]][...]
```

Comment:
I would rather use `s0` here. What you are doing might have some assumptions on key ordering.

Reply:
Indeed, here I was not sure what was the intended behavior..  Will it always be `s0` or should we expect multiple versions? I could not find any stack with multiple `s*`.. wouldn't be better to use something like `re` to always pick the dataset with the highest count?

Reply:
So the way the multi-resolution format works is like this:
```
/data
  /s0 - highest resolution data-set
  /s1 - first downsampled version of the data-set (usually by a factor of 2)
 /s2 - second downsampled version of the data-set (usually by a factor of 4 w.r.t s0)
 ....
```
So `s0` is guranteed to be there and it's usually what you'd want to use.

Reply:
Ah now I see.. I thought those where independent version of the same "semantic" key. It's all fixed now